astroquery/mpc/tests/test_mpc.py

```diff
@@ -176,19 +176,6 @@ def test_get_ephemeris_Moon_phase(patch_post):
     assert result['Moon phase'][0] >= 0
 
 
-def test_get_ephemeris_Uncertainty(patch_post):
```

Comment:
what are the reason for removing these tests?

Reply:
Ah, this is a mistake.  I thought I had made a copy-paste error, but these were actually working mocked tests.  I can restore them.

Reply:
I partially thought it was a mistake because `test_get_ephemeris_Moon_phase_and_Uncertainty` was defined twice.

I think PR #3019 attempted to replace 1994 XG with 2024 AA, even though in the mocked tests it wasn't needed (because the mocked data didn't change, only the online results).  I'll clean this up.